src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap

```diff
@@ -296,3 +296,332 @@ exports[`PackageConfirmation Component Matches the snapshot 1`] = `
   </div>
 </DocumentFragment>
 `;
+
+exports[`PackageConfirmation Component Matches the new account snapshot 1`] = `
+<DocumentFragment>
+  <div
+    class="page"
+  >
+    <div
+      class="content col-md-8"
+    >
+      <div
+        class="display-box success-background"
+      >
+        <div
+          class="display-icon"
+        >
+          <div
+            style="color: rgb(46, 133, 64);"
+          >
+            <svg
+              fill="currentColor"
+              height="32"
+              size="32"
+              stroke="currentColor"
+              stroke-width="0"
+              viewBox="0 0 24 24"
+              width="32"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
+              />
+            </svg>
+          </div>
+        </div>
+        <div
+          class="display-element"
+        >
+          CSO Account created successfully.
```

Comment:
Your CSO Account?

Reply:
I like this, can check with julia and then make the change if she's good with it